website/docs/index.html.markdown

```diff
@@ -189,7 +191,7 @@ For more information on configuring and managing programmatic API Keys see the [
 | 1.4.x | 2023-03-08 | 2025-03-31 | 2025-03-31 |
 | 1.3.x | 2022-09-21 | 2024-09-30 | 2024-09-30 |
 | 1.2.x | 2022-05-18 | 2024-05-31 | 2024-05-31 |
-
+<!-- MATRIX_PLACEHOLDER_END -->
```

Comment:
a nice test would be to have this empty in the PR as see how tomorrow the PR is generated to create the table

Reply:
I tried running the job manually in the fork and PR was created https://github.com/maastha/terraform-provider-mongodbatlas-fork/pull/1
WDYT?